ledger-core-ethereum/src/ethereum/EthereumLikeTransaction.cpp

```diff
@@ -66,7 +66,7 @@ namespace ledger {
 
             _gasPrice = std::make_shared<Amount>(_currency, 0, tx.gasPrice);
             _gasLimit = std::make_shared<Amount>(_currency, 0, tx.gasLimit);
-            _gasUsed = std::make_shared<Amount>(_currency, 0, tx.gasUsed.getValue());
+            _gasUsed = std::make_shared<Amount>(_currency, 0, tx.gasUsed.getValueOr(BigInt::ZERO));
```

Comment:
So this fix is required because now we use constructors instead of dummy initializing objects (like with `Foo{}`).